test/integration/controller/admissionchecks/provisioning/provisioning_test.go

```diff
@@ -607,24 +630,15 @@ var _ = ginkgo.Describe("Provisioning", ginkgo.Ordered, ginkgo.ContinueOnFailure
 			})
 
 			ginkgo.By("Checking provision request is deleted", func() {
-				provReqKey := types.NamespacedName{
-					Namespace: wlKey.Namespace,
-					Name:      provisioning.GetProvisioningRequestName(wlKey.Name, ac.Name, 1),
-				}
 				gomega.Eventually(func() error {
-					request := &autoscaling.ProvisioningRequest{}
-					return k8sClient.Get(ctx, provReqKey, request)
+					return k8sClient.Get(ctx, provReqKey, createdRequest)
 				}, util.Timeout, util.Interval).Should(testing.BeNotFoundError())
 			})
 
 			ginkgo.By("Checking the workload remains admitted", func() {
```

Comment:
I just realized this suite does not run the workload controller. In that case we would really not expect eviction. So, I think what we should test here is:
1. that the admission check remains in the Ready state
2. the Provisioning request is not recreated

Reply:
Should I remove `"Checking the workload remains admitted"`?

Reply:
Good question. I think it is ok to drop it. Maybe just add a comment next to the new one: "Checking the workload admission check remains Ready" saying "We use this as a proxy check to verify that the workload remains admitted, because the test suite does not run the workload controller".

Reply:
OK, Thank you!

Reply:
Alternatively we could consider enabling the workload controller in a new dedicated suite, but this feels overkill for this PR.